tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -204,7 +204,7 @@ spec:
             os="$(jq -rs 'map(.platform.os) | .[0]' <<< "$arch_json")"
             build_date="$(skopeo inspect --no-tags --override-os "${os}" --override-arch "${arch}" \
               docker://"${IMAGE_REF}" | jq -r '.Labels."build-date"')"
-            timestamp="$(date -d "${build_date}" "+$passedTimestampFormat")"
+            timestamp="$(date -d "${build_date}" "+$passedTimestampFormat" || true)"
             if [ "${build_date}" == "null" ] ; then
```

Comment:
I was looking at this today (while working on [RELEASE-1151](https://issues.redhat.com/browse/RELEASE-1151)) and I don't get this logic. I am gonna propose to change it to this:
```sh
            build_date="$(jq -r '."build-date"' <<< "$labels")"
            if [ "${build_date}" = "null" ] ; then
              timestamp=""
            else
              timestamp="$(date -d "${build_date}" "+$passedTimestampFormat")"
            fi
```
(And then if `{{ timestamp }}` is used, it would fail because of the check for empty variable expansion.)

Reply:
```
$ mytime=$(date -d null +%s || true)
date: invalid date ‘null’
$ echo $?
0
$ mytime=$(date -d null +%s)
date: invalid date ‘null’
$ echo $?
1
```

Reply:
Ha, thanks. So that's good to remember - if you use `$()` in arguments of another command, then it won't fail it. But if you use it in a variable assignment, it can fail it.

Reply:
All good to rework it in your PR though. This one was a quick fix to unblock a user